core/shared/src/test/scala/JwtSpec.scala

```diff
@@ -40,7 +40,7 @@ class JwtSpec extends munit.FunSuite with Fixture {
     dataRSA.foreach { d => battleTestEncode(d, privateKeyRSA, validTimeJwt) }
   }
 
-  test("should encode Ed25519") {
+  test("should encode EdDSA") {
     dataEdDSA.foreach { d => battleTestEncode(d, privateKeyEd25519, validTimeJwt) }
```

Comment:
I haven't renamed the Ed25519 private key name, since it is an Ed25519 private key, and uses the Ed25519 private key generator.